ha/cli/command_factory.py

```diff
@@ -73,6 +73,11 @@ def __init__(self):
                 "-h": "ha.cli.exec.commandExecutor.ServiceCLIUsage",
                 "--help": "ha.cli.exec.commandExecutor.ServiceCLIUsage"
             },
+            "support_bundle": {
```

Comment:
Please comment on if it was decided to have this as a CLI.

Reply:
This is a pre-existing CLI from 1.0 
I have ensured that the same is present here

Reply:
@ajaysrivas So this will be CLI, is it?

Reply:
Fine with the CLI.